src/shared/services/configuration/map-options.ts

```diff
@@ -4,10 +4,11 @@ import type { MapOptions } from 'leaflet';
 import configuration from 'shared/services/configuration/configuration';
 
 const MAP_OPTIONS: MapOptions = {
-  ...(configuration?.map.options as MapOptions),
+  ...(configuration.map.options as MapOptions),
   attributionControl: true,
   crs: getCrsRd(),
   zoomControl: false,
+  dragging: !('ontouchstart' in window),
```

Comment:
Maybe an comment here that this works on mobile but might not work on DevTools emulator?